File: atividade4/lista_compras.py
itens = []
precos = []

quantidade = int(input("Quantos itens deseja adicionar? "))

for i in range(quantidade):
    item = input("Digite o nome do item: ")
    preco = float(input("Digite o preço do item: "))

    itens.append(item)
    precos.append(preco)

total = 0

for i in range(quantidade):
    total = total + precos[i]

print("\n--- RESUMO DA COMPRA ---")

for i in range(quantidade):
    print(itens[i], "- R$", precos[i])

print("Total: R$", total)

File: atividade4/ListaCompras.java
import java.util.Scanner;

public class ListaCompras {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.print("Quantos itens deseja adicionar? ");
        int quantidade = scanner.nextInt();
        scanner.nextLine();

        String[] itens = new String[quantidade];
        double[] precos = new double[quantidade];

        for (int i = 0; i < quantidade; i++) {

            System.out.print("Digite o nome do item: ");
            itens[i] = scanner.nextLine();

            System.out.print("Digite o preço do item: ");
            precos[i] = scanner.nextDouble();
            scanner.nextLine();
        }

        double total = 0;

        for (int i = 0; i < quantidade; i++) {
            total = total + precos[i];
        }

        System.out.println();
        System.out.println("--- RESUMO DA COMPRA ---");

        for (int i = 0; i < quantidade; i++) {
            System.out.printf("%s - R$ %.2f%n", itens[i], precos[i]);
        }

        System.out.printf("Total: R$ %.2f%n", total);

        scanner.close();
    }
}